Create issue template to report bugs (#12)
primer/prism: .github/ISSUE_TEMPLATE/prism-bug-report.md

---

> name: Prism bug report

Reviewer: ```suggestion
name: Bug report
```

> about: Create a report to help us improve Prism

Reviewer: ```suggestion
about: Help us improve Prism by reporting a bug
```
